docs/buildah-unshare.md

```diff
@@ -19,6 +19,14 @@ manually clearing storage and other data related to images and containers.
 It is also useful if you want to use the `buildah mount` command.  If an unprivileged users wants to mount and work with a container, then they need to execute
 buildah unshare.  Executing `buildah mount` fails for unprivileged users unless the user is running inside a `buildah unshare` session.
 
+## OPTIONS
+**--mount** [*VARIABLE=]containerNameOrID*
+
+Mount the *containerNameOrID* container while running *command*, and set the
```

Comment:
Suggest:  `Mount the container specified by the *containerNameOrID* while running ....`

Reply:
actually drop the 'the'
`Mount the container specified by *containerNameOrID* while running ....`